pkg/azureutils/azure_disk_utils.go

```diff
@@ -117,6 +117,14 @@ const (
 	Uncached
 )
 
+type CRIUpdateMode uint
+
+const (
+	UpdateCRI       CRIUpdateMode = 0001
```

Comment:
In Golang, an [integer literal](https://go.dev/ref/spec#Integer_literals) with a leading `0` indicates an octal number. Given you're use of '0' & '1' here, I suspect you wanted binary? If so, you need to prefix with `0b`.